third_party/rn/react_native.BUILD

```diff
@@ -18,6 +18,9 @@ cmake(
     generate_args = ["-G Ninja"],
     lib_name = "jsi",
     lib_source = "jsi_sources",
+    env = {
```

Comment:
i don't think this is even being used right now

Reply:
Like this step at all?

Reply:
no harm leaving it in though, probably eventually get to use this, but brings up this point
> could we pull this out to config set by command line, or set by something in .bazelrc

when we remove it after upgrade we just remove it in one place

Reply:
I don't think we can, I wasn't able to find a way to set args for certain targets via the CLI. If it was a bit harder to find and untangle I would agree but since this is pretty easy to search and remove I am not too worried about it.